Maple2.Server.Game/PacketHandlers/ItemUseHandler.cs

```diff
@@ -86,4 +90,20 @@ public class ItemUseHandler : PacketHandler<GameSession> {
 
         session.Send(ChatStickerPacket.Add(item, new ChatSticker(stickerSetId, session.Player.Value.Unlock.StickerSets[stickerSetId])));
     }
+
+    private static void HandleTitleScroll(GameSession session, Item item) {
+        if (!int.TryParse(item.Metadata.Function?.Parameters, out int titleId)) {
+            return;
+        }
+
+        if (session.Player.Value.Unlock.Titles.Contains(titleId)) {
+            session.Send(ChatPacket.Alert(StringCode.s_title_scroll_duplicate_err));
+            return;
+        }
+
+        session.Player.Value.Unlock.Titles.Add(titleId);
```

Comment:
This should also happen after consuming, otherwise you could get a title without using the scroll or whatever.